src/__tests__/to-be-invalid.js

```diff
@@ -0,0 +1,72 @@
+import {JSDOM} from 'jsdom'
+import {render} from './helpers/test-utils'
+
+test('.toBeInvalid', () => {
+  const {queryByTestId} = render(`
+    <div>
+      <input data-testid="no-aria-invalid">
+      <input data-testid="aria-invalid" aria-invalid>
+      <input data-testid="aria-invalid-value" aria-invalid="true">
+      <input data-testid="aria-invalid-false" aria-invalid="false">
+    </div>
+    `)
+
+  const dom = new JSDOM(`<input pattern="[a-z]{1,3}" value="test+">`)
```

Comment:
I got your comment about this. I do not like it too much, but if putting this above did not work we're have to live with it. Maybe add a one-liner comment above this line with a `TODO:` and a link to the text where you explained this in your PR, or an explanation here, for future context if we can ever fix it.